restart_process/Tiltfile

```diff
@@ -40,29 +40,36 @@ def docker_build_with_restart(ref, context, entrypoint, live_update,
     base_ref = '{}{}'.format(ref, base_suffix)
     docker_build(base_ref, context, **kwargs)
 
-    # declare a new docker build that adds a static binary of entr (previously
-    # compiled by tilt) to the user's image
+    # declare a new docker build that adds a static binary of tilt-restart-wrapper
+    # (which makes use of entr to watch files and restart processes) to the user's image
     df = '''
-    FROM tiltdev/entr:2020-16-04 as entr-img
```

Comment:
lol, oops

Reply:
You mean we're not 16 months into the pandemic?